src/version.cpp.in

```diff
@@ -1,7 +1,25 @@
-#include <string>
+#include <morphio/version.h>
 
-#include <morphio/types.h>
+#include <string>
 namespace morphio
 {
-  const std::string VERSION("@MORPHIO_VERSION_STRING@");
-}
+    std::string getVersionString() {
```

Comment:
Could this actually be `constexpr` in the latest versions of C++?

Reply:
if memory serves, constexpr doesn't work w/ std::string; something to do w/ the ctor or dtor.

Reply:
`std::string_view` is what I've seen floating around for this purpose. Anyways, I'm fine with it as it is.

Reply:
That's a c++17 thing, and forcing consumers of morphio to be that 'modern' is challenging, I'd say.  In addition, there are some fun gotchas w/ string_view that are 'fun', so it's one of the features that I'm not comfortable using until there is a strong consensus on when/where to use it.

Reply:
Who are these consumers? But yes, we can hold off on all that.

Reply:
Since it's open source, we don't really know who all uses it.

However, the plan/hope is to have NeuroM use this as the morphology reading backend, which means that we have to be able to build it on Windows and macos.  The latter, afaik, uses libc++, which doesn't have complete c++17 support.  Correct me if I'm wrong, though.

Reply:
It's still software from us that's using it? I normally look at contributors and people opening issues to see who's using it…

Reply:
Fair enough; the thing is that if NeuroM is to use this, we need to support the platforms mentioned above.  The people who use it, from what I can tell, aren't software engineers, and don't do a lot of github interaction.  JDC has looked at the stats, for instance, and many (most?) of the users of NeuroM are on windows.  They don't, however, seem to open many issues.